build.gradle

```diff
@@ -56,6 +57,10 @@ sourceSets {
         scala {
             srcDirs = ['main/src']
         }
+
+        resources {
+            srcDirs = ['main/src/resources']
+        }
```

Comment:
IntelliJ seems to be struggling with this, but Gradle handles it fine.

Reply:
OK- that is acceptable.